### ISSUE TITLE
Allow to display images using the original generated thumnail size

### ISSUE THREAD

**Issue by cryptomilk** [thenadz/document-gallery#33]
When I create a document gallery I would like to show images in thumbnail size they got created without any scaling.

In the Settings for Document Gallery you allow to specify: Max Thumbnail Dimensions.

I've changed them to create thumbnails for PDF files with a width of 250px. However by default they are displayed scaled down with a width of: 89px

```
.document-gallery .document-icon img {
    width: 89px;
    max-width: 100%;
    border: none;
    margin: 0 auto;
}
```

So that I get the thumbnail size I specified, I needed to set a Custom CSS.

```
.document-gallery .document-icon img {
    width: 250px;
    max-width: 100%;
    border: 0;
    margin: 0 auto;
}
```

It would be really nice to have an option which I could select when creating a Document Gallery to use the original thumbnail size without any scaling.

Example with bigger thumbnail size:
https://ig-klettern.org/verein/steinschlag/

**Comment by thenadz** [comment]
@cryptomilk I completely agree. The DG CSS is in desperate need of an overhaul and the issue you bring up has come up more times than I can count. I will try to get a change in soon!
